db/migrate/20190510082811_add_taxonomy_id_taxable_type_indices_to_taxable_taxonomies.rb

```diff
@@ -0,0 +1,7 @@
+class AddTaxonomyIdTaxableTypeIndicesToTaxableTaxonomies < ActiveRecord::Migration[5.2]
+  def change
+    change_table :taxable_taxonomies do |t|
+      t.index ["taxonomy_id", "taxable_type"], name: "index_taxable_taxonomy_id_type"
```

Comment:
Is the name needed? rails auto generates one that it knows to remove later.

Reply:
Yeah I've seen it in previous migrations so I thought it's our convention. Let's drop it then.